package.json

```diff
@@ -4,6 +4,7 @@
   "homepage": "https://github.com/marionettejs/backbone.radio",
   "version": "2.0.0",
   "main": "build/backbone.radio.js",
+  "module": "build/backbone.radio.esm.js",
```

Comment:
Does this file need to be checked into the source tree or is it automatically generated?